File: app/src/main/java/ru/neverlands/abclient/postfilter/ChRoomPhp.java
package ru.neverlands.abclient.postfilter;

/**
 * Заглушка пост‑фильтра для ch/room.php.
 * В ПК версии здесь есть логика обработки комнаты; на Android пока не требуется.
 * Оставляем как точку расширения для будущего портирования.
 */
public class ChRoomPhp {
    /**
     * Возвращает исходный ответ без изменений.
     */
    public static byte[] process(byte[] array) {
        return array;
    }
}

File: ANClient/PostFilter/ChRoomPhp.cs
using System.IO;

namespace ANClient.PostFilter
{
    using Helpers;

    internal static partial class Filter
    {
        private static byte[] ChRoomPhp(byte[] array)
        {
            /*
            string html;            
            try
            {
                html = File.ReadAllText("anclient_test_ch.php.txt");
            }
            catch (FileNotFoundException)
            {
                html = Russian.Codepage.GetString(array);
            }
            */

            var html = Russian.Codepage.GetString(array);
            html = RoomManager.Process(html);
            return Russian.Codepage.GetBytes(html);
        }
    }
}